### PR TITLE
Add fragrance free consideration

### PR DESCRIPTION
**Pull request by Mariatta**

### CODE REVIEW

File: content/accessibility/contents.lr [hunk 1]
```diff
@@ -99,3 +99,17 @@ Mother's room
 --------------
 
 While there is not a dedicated mother's room in Kane Hall, we do have access to a gender-neutral single occupant restroom and two locking changing rooms. These are all located in the green room. Please ask any member of staff and they can show you how to find it. You can also reach out to us in advance to ensure you have access.
+
+
+Fragrance free consideration
+-----------------------------
+
+We would like to ask that you kindly consider the health and well-being of your fellow attendees’. If you are able,
```
Comment (ellezv) [inline content/accessibility/contents.lr]:
```suggestion
We would like to ask that you kindly consider the health and well-being of your fellow attendees. If you are able,
```

File: content/accessibility/contents.lr [hunk 2]
```diff
@@ -99,3 +99,17 @@ Mother's room
...
+please refrain from wearing and/or spraying chemicals such as perfumes, cologne, scented products in the conference space.
+
+This consideration will help to ensure that our space is accessible and available to all members of our community–including
+those who have Multiple Chemical Sensitivities (MCS) or chemical injury. The health effects of fragrances – ranging in
+severity from mild to debilitating – can include: headaches, migraines, dizziness, nausea, fatigue, shortness of breath
```
Comment (ellezv) [inline content/accessibility/contents.lr]:
```suggestion
severity from mild to debilitating.
```

Also it looks like sentences are on multilines. I think paragraphs should be in one line.

Comment (Mariatta) [reply content/accessibility/contents.lr]: Thanks! i made the change, you can see it here at the very bottom https://deploy-preview-64--2019-pycascades.netlify.com/accessibility/

File: content/accessibility/contents.lr [hunk 3]
```diff
@@ -99,3 +99,17 @@ Mother's room
...
+Fragrance free consideration
+-----------------------------
+
+We would like to ask that you kindly consider the health and well-being of your fellow attendees. If you are able,
+please refrain from wearing and/or spraying chemicals such as perfumes, cologne, scented products in the conference space.
+
+This consideration will help to ensure that our space is accessible and available to all members of our community–including
+those who have Multiple Chemical Sensitivities (MCS) or chemical injury. The health effects of fragrances – ranging in
+severity from mild to debilitating.
+and wheezing. For some people, especially those with asthma and other respiratory disorders, exposure to fragrances can
```
Comment (ellezv) [inline content/accessibility/contents.lr]:
```suggestion
 For some people, especially those with asthma and other respiratory disorders, exposure to fragrances can
```

**Pull request merged**